package.json

```diff
@@ -19,12 +19,13 @@
     "test-cypress": "cypress open",
     "test-cypress-build": "cypress run --spec ./cypress/features/build/build*.feature --browser chrome --reporter spec",
     "test-cypress-accessibility": "cypress run --spec ./cypress/features/accessibility/accessibility.feature --browser chrome --reporter spec",
-    "test-cypress-regression": "npm run cypress-group-regression-common; npm run cypress-group-regression-experimental; npm run cypress-group-regression-deprecated; npm run cypress-group-regression-validations; npm run cypress-group-regression-themes; npm run generate-cypress-allure-report",
-    "cypress-group-regression-common": "cypress run --spec './cypress/features/regression/*.feature' --reporter mocha-allure-reporter --browser chrome",
-    "cypress-group-regression-experimental": "cypress run --spec './cypress/features/regression/experimental/*.feature' --reporter mocha-allure-reporter --browser chrome",
-    "cypress-group-regression-deprecated": "cypress run --spec './cypress/features/regression/deprecated/*.feature' --reporter mocha-allure-reporter --browser chrome",
-    "cypress-group-regression-validations": "cypress run --spec './cypress/features/regression/validations/*.feature' --reporter mocha-allure-reporter --browser chrome",
-    "cypress-group-regression-themes": "cypress run --spec './cypress/features/themes/themes.feature' --reporter mocha-allure-reporter --browser chrome",
+    "test-cypress-regression": "npm run test-cypress-accessibility; npm run cypress-group-regression-validations; npm run cypress-group-regression-themes; npm run cypress-group-regression-common; npm run cypress-group-regression-experimental; npm run cypress-group-regression-deprecated",
+    "cypress-group-regression-common": "cypress run --spec './cypress/features/regression/*.feature' --reporter spec --browser chrome",
+    "cypress-group-regression-experimental": "cypress run  --spec './cypress/features/regression/experimental/*.feature' --reporter spec --browser chrome",
```

Comment:
`cypress-group-regression-*` are not used by anything, we should remove these. Even running locally you can use the vscode launcher to run a specific cypress file.

Reply:
`test-cypress-regression` is using all those `cypress-group-regression-*` scripts